ensime_shared/ensime.py

```diff
@@ -130,13 +130,14 @@ def disable_plugin(self):
             If used from a secondary thread, this may need to use threadsafe
             Vim calls where available -- see :meth:`Editor.raw_message`.
         """
-        for path in self.runtime_paths:
+        for path in self.runtime_paths():
             self._vim.command('set runtimepath-={}'.format(path))
 
-    @property
+    # Tried making this a @property, with and without memoization, and it made
+    # plugin initialization misbehave in Neovim (only). WTF.
     def runtime_paths(self):  # TODO: memoize
         """All the runtime paths of ensime-vim plugin files."""
-        runtimepath = self._vim.eval('&runtimepath')
+        runtimepath = self._vim.options['runtimepath']
```

Comment:
This didn't actually solve it, I tried both, the critical part was the method can't be a `@property`. Still don't know why, because `using_server_v2` has seemed to work fine as a property on this class.

You should see the huge WIP I've finally made progress on that follows after this 😁 No more launcher object in `EnsimeClient` either. Going to need a lot of testing because it's hard to make the changes small, but once it's robust it should hopefully make stuff like the server restarting way easier. Definitely need the build sorted out when that lands…